### PR TITLE
#157 style meeting update component conten

### PR DESCRIPTION
**Pull request by GALTdea**
I have updated the code to render the user's name above the content. However, I am submitting a draft Pull Request because there is an issue with the columns. Specifically, when one of the columns has no content, it fails to maintain its width. 

 
Screen Shot 2023-07-11 at 9 08 42 PM

with daisyui line
Screen Shot 2023-07-13 at 3 39 41 PM

### CODE REVIEW

**Comment by toyhammered** [comment]
Screenshot 2023-07-11 at 9 41 39 PM
I just pulled it down and it seems ok?

File: app/components/standup_meeting/meeting_update_component.html.erb [hunk 1]
```diff
@@ -1,5 +1,8 @@
-<div class="border p-4 flex flex-col lg:flex-row justify-between mt-4">
-  <span class="flex flex-col justify-center mr-4 font-bold"><%= user.email %></span>
+<div class="border p-4 flex flex-col  justify-between mt-4">
+  <span class="flex flex-col justify-center mr-4 font-bold">
+    <%= user.full_name %>
+  </span>
+  <hr class="border-t border-gray-400 my-4" />
```
Comment (afogel) [inline app/components/standup_meeting/meeting_update_component.html.erb:5]: nit, but so you should know -- you can use the native daisy class:
`<div class="divider my-4 "></div>`

**Review by afogel — COMMENTED** [review]
Overall, lgtm! re: formatting, as daniel mentioned, there looks to be an issue on your local machine re: daisy styling. I'd try to clobber your assets, make sure you're yarn installed and bundled, restart your server, and see whether that improves things.

File: app/components/standup_meeting/meeting_update_component.html.erb (hunk 1 above)
Comment (GALTdea) [reply app/components/standup_meeting/meeting_update_component.html.erb]: done

**Comment by GALTdea** [comment]
ok added the daisyui line? let me know what you think @toyhammered @afogel

**Review by toyhammered — APPROVED** [review]

**Pull request merged**